raiden/tests/unit/test_raiden_event_handler.py

```diff
@@ -36,39 +39,40 @@ def test_handle_contract_send_channelunlock_already_unlocked():
         token_network_address=token_network_address,
         partner_address=participant,
     )
+    assert channel_state
 
     channel_state.our_state.onchain_locksroot = LOCKSROOT_OF_NO_LOCKS
     channel_state.partner_state.onchain_locksroot = LOCKSROOT_OF_NO_LOCKS
 
     def detail_participants(  # pylint: disable=unused-argument
-        participant1, participant2, block_identifier, channel_identifier
+        _participant1, _participant2, _block_identifier, _channel_identifier
     ):
-        transferred_amount = 1
-        locked_amount = 1
-        locksroot = make_32bytes()
+        transferred_amount = TA(1)
+        locked_amount = TA(1)
+        locksroot = make_locksroot()
         balance_hash = hash_balance_data(transferred_amount, locked_amount, locksroot)
         our_details = ParticipantDetails(
             address=raiden.address,
-            deposit=5,
-            withdrawn=0,
+            deposit=TA(5),
+            withdrawn=WA(0),
             is_closer=False,
             balance_hash=balance_hash,
-            nonce=1,
+            nonce=Nonce(1),
             locksroot=locksroot,
             locked_amount=locked_amount,
         )
 
-        transferred_amount = 1
-        locked_amount = 1
+        transferred_amount = TA(1)
```

Comment:
I can see why you use abbreviations ... but I kind of dislike them. And it's not consistent with the rest of the codebase. Can you maybe use the full names? There are some places where the cast is done 5-6 times in one line where this would make sense but this here is not required I think.